DevoxxClientMobile/src/main/resources/com/devoxx/common.css

```diff
@@ -249,17 +249,22 @@
     -fx-font-size: 1.05em;
     -fx-wrap-text: false;
 }
+
+.schedule-cell > .list-tile > .text-box > .extra-text,
```

Comment:
I'd probably use a different styling for the extra text. With so many lines now, it is hard to read.

Reply:
This is how the cell looks with the extra label at the bottom:

![image](https://user-images.githubusercontent.com/3197675/42617760-66b535b4-85d0-11e8-8496-47d27a6b4ba2.png)

Reply:
With different styling do you want to use a different color for the text or alter the text size?

Reply:
Personally, I'd use lighter color and smaller font size.

Reply:
How about this?

![image](https://user-images.githubusercontent.com/3197675/42619878-edb48e24-85d6-11e8-8e2c-1b27e6280eca.png)

Reply:
Looks fine to me.